src/main/webapp/filter.js

```diff
@@ -55,11 +56,29 @@ function sendClickedFiltersToServer() { //eslint-disable-line
   }
   const jsonDiffArray = JSON.stringify(clickedDifficulties);
 
-  // TODO: Get clicked num stops
+  // Get clicked num stops
+  let clickedNumStops = [];
+  let numStops = '';
+  if (document.getElementsByClassName('num-stops').length > 0) {
+    const numStopsContainer = document.getElementsByClassName('num-stops')[0];
+    clickedNumStops =
+        Array.from(numStopsContainer.getElementsByClassName(CLICKED))
+            .map((element) => element.innerText);
+    if (clickedNumStops.length > 1) {
+      window.alert('Please choose only ONE number of stops.');
```

Comment:
When this happens, should the method return so that the request to the server is not made?

Reply:
I think that you should enforce this in the UI (i.e. only allow them to click one of the numbers, not both 3 and 5) instead of letting them click anything and then triggering an alert.

I'm also curious why it's only 3 or 5?  The number seems arbitrary.  Why can't they select any number?